jobs/rocoto/aeroanlfinal.sh

```diff
@@ -1,12 +1,19 @@
 #! /usr/bin/env bash
-
-source "$HOMEgfs/ush/preamble.sh"
+export STRICT="NO"
+source "${HOMEgfs}/ush/preamble.sh"
 
 ###############################################################
-# Source FV3GFS workflow modules
-. $HOMEgfs/ush/load_fv3gfs_modules.sh
+# Source UFSDA workflow modules
+. "${HOMEgfs}/ush/load_ufsda_modules.sh"
 status=$?
-[[ $status -ne 0 ]] && exit $status
+[[ ${status} -ne 0 ]] && exit "${status}"
```

Comment:
Please add lines:
```
export job="aeroanlfinal"
export jobid="${job}.$$"

repeat for the other 2 rocoto jobs appropriately.

Reply:
merging in develop should take care of this.